kanidm_unix_int/Cargo.toml

```diff
@@ -70,6 +70,7 @@ tokio-util = { workspace = true, features = ["codec"] }
 tracing.workspace = true
 reqwest.workspace = true
 users.workspace = true
+walkdir = "2"
```

Comment:
should be walkdir.workspace = true, and then in the top level Cargo.toml set in workspace.dependencies walkdir = "2"